test/source_code_test.rb

```diff
@@ -80,7 +80,6 @@ def self.token_is_keyword?(str, lex, index, token)
               %r{engines/bastion/test/test_helper\.rb},
               %r{test/support/vcr\.rb},
               %r{test/support/runcible\.rb},
-              %r{test/katello_test_runner\.rb}, # TODO: clean up minitest_helper
```

Comment:
@ehelms for that particular failure you mentioned, it seems like this change was not there when you ran the test?